webservice/webservice.py

```diff
@@ -322,7 +326,64 @@ def get_manifest():
     # Get the response back
     logger.info("Creating the API response")
     response = es_td.transform_manifest(filters=filters)
-    # Return the excel file
+    if output_formats == 'tsv':  # returns metadata as TSV file
+        return response
+    elif output_formats == 'bdbag':  # returns presigned URL to a BDBag in S3
+        response_bag = bdbag_response(response)
+        return jsonify(response_bag)
+
+
+def bdbag_response(response_obj):
+    logger = logging.getLogger('dashboardService.webservice.bdbag_response')
+    # Create and return the BDbag folder.
+    bag_name = 'manifest'
+    bag_info = {'organization': 'UCSC Genomics Institute',
+                'data_type': 'TOPMed',
+                'date_created': datetime.datetime.now().isoformat()}
+
+    # Instantiate bag object.
+    bag = BagHandler(data=response_obj.get_data(),
+                     bag_info=bag_info,
+                     bag_name=bag_name)
+    zipped_bag = bag.create_bag()  # path to compressed bag
+    logger.info("Creating a compressed BDbag containing manifest.")
+
+    # Import bucket environment variable, launch instance of S3-file handler,
+    # and upload the BDBag file to S3.
+
+    # Transfer parameters.
+    aws_region = 'us-west-2'
```

Comment:
Jesse added a parameter for the AWS region. You should use that instead of hard-coding it.

Reply:
Ah, that's great. I meant to add it because it's needed here. Glad it's done already.